app/src/processing/app/Mode.java

```diff
@@ -565,77 +565,6 @@ public void actionPerformed(ActionEvent e) {
     }
   }
 
-
```

Comment:
My bad, I thought I was just cleaning up the file. Commented out code can be very dangerous because when code changes the commented out code may not work anymore. 

I've noticed that a lot of PDE code has lots of random code commented out.